src/ResourceManager/Profile/Commands.Profile/Context/SaveAzureRMContext.cs

```diff
@@ -65,7 +67,7 @@ public override void ExecuteCmdlet()
                         throw new ArgumentException(Resources.AzureProfileMustNotBeNull);
                     }
 
-                    if (!AzureSession.DataStore.FileExists(Path) || Force ||
+                    if (!AzureSession.DataStore.FileExists(Path) || Force.IsPresent ||
```

Comment:
Well, because there is only one parameter set and File is always provided.  We are nit checking for a null path, we are checking for file existence to prompt the user for overwrite.

Why would I want to avoid verbose messages? The verbose message is entirely appropriate - it is printed if the user is printing more detailed messages, and is on the verbose stream regardelsss.  

The cmdlet id designed to return nothing ot throw, this is a typical PowerShell pattern for action cmdlets.  Any change in the output would be a breakign change in any case.